main.py

```diff
@@ -87,17 +107,17 @@ async def newHeader(message, *args):
           await message.channel.send(f'You can\'t make two headers of the same name. \n(if ur trying to find bugs id like you to know ur an asshole)')
           
         else:
-          spreadSheet.write_cell(f'{string.ascii_uppercase[count]}1', headers)
+          self.spreadsheet.write_cell(f'{string.ascii_uppercase[count]}1', headers)
```

Comment:
WARNING: this implementation is successful if and only if `count` is within the bounds of `ascii_uppercase`, i.e. between 0 and 25. We should either (a) remove this limitation and enable spreadsheets with more than 25 headers (I can rewrite the formula to allow this) or (b) do index-checking before running this else-case, and print an error to the user in an effort to fail gracefully.
@megumin00 which do you prefer?